config/minimalist-templates/popups_templates.yaml

```diff
@@ -0,0 +1,531 @@
+                - ulm_language_variables
+              entity: "[[[ return entity.entity_id ]]]"
+              show_label: none
+          item2:
+            card:
+              type: custom:state-switch
+              entity: hash
+              default: color
+              transition: flip
+              transition_time: 300
+              states:
+                brightness:
+                  type: custom:button-card
+                  styles:
+                    grid:
+                      - grid-template-areas: '"item1"'
+                      - grid-template-columns: 1fr
+                      - grid-template-rows: min-content
+                    card:
+                      - background: none
+                      - box-shadow: none
+                      - padding: 4%
+                    # custom_fields:
+                    #   item2:
+                    #     - background-color: blue
+                    #     - border-radius: 12px
+                    #     - position: absolute
+                    #     - left: 64%
+                    #     - top: >
+                    #         [[[
+                    #           var bri = states[entity.entity_id].attributes.brightness;
+                    #           return Math.round(bri ? (80-(72*bri/256)) : '80') + '%';
+                    #         ]]]
+                    #     - height: 25px
+                    #     - width: 45px
+                    #     - font-size: 11px
+                    #     - line-height: 25px
+                    #     - color: white
+                  custom_fields:
+                    item1:
+                      card:
+                        type: "custom:slider-card"
+                        entity: "[[[ return entity.entity_id ]]]"
+                        radius: 20px
+                        rotate: 270
+                        containerHeight: 320px
+                        width: 250px
+                        height: 100px
+                        top: 100px
+                        left: 17%
+                        thumbHorizontalPadding: 0px
+                        thumbWidth: 0px
+                        mainSliderColor: rgba(var(--color-theme),0.1)
+                        mainSliderColorOff: rgba(var(--color-theme),0.1)
+                        secondarySliderColor: var(--color-theme)
+                        secondarySliderColorOff: var(--color-theme)
+                        card_mod:
+                          style: |
+                            ha-card {
+                              background: none !important;
+                              box-shadow: none !important;
+                            }
+                            .off {
+                              border: 1px solid rgba(var(--color-theme),0.3) !important;
+                            }
+                            .on {
+                              border: 1px solid rgba(var(--color-theme),0.3) !important;
+                            }
```

Comment:
I have added a flexbox to the slider card because it was not centered in most of my screens
```
.slider-container {
    display: flex;
    justify-content: center;
    align-items: center;
}
```

And in the slider-container input from above:
```
.slider-container input[type="range"]{
   position: revert !important;
}
```